linux/utils/enable_vmtools_logging.yml

```diff
@@ -22,10 +22,12 @@
 # VMware Tools config file on FreeBSD with open-vm-tools 10.1.x or later
 - name: "Set fact of VMware Tools config file for FreeBSD"
   ansible.builtin.set_fact:
-    vmtools_config_file: "/usr/local/share/vmware-tools/tools.conf"
+    vmtools_config_file: |-
+      {%- if linux_ovt_install_type is defined and linux_ovt_install_type == 'source' -%}/etc/vmware-tools/tools.conf
```

Comment:
Please use `vmtools_install_type` instead of `linux_ovt_install_type` as `vmtools_install_type` will always be set but `linux_ovt_install_type` not.

You can just change the when at line 19~20 and here. VMTools 10.1.0 is deprecated. You can replace the condition about checking vmtools_version with checking `vmtools_install_type`.

For example, line 20 can be replaced by: `vmtools_install_type != 'package'`. And add `vmtools_install_type == 'package'` for this task.

Reply:
Is it OK for two tasks: one for linux and other for FreeBSD ?